charts/kube-otel-stack/values.yaml

```diff
@@ -348,7 +348,7 @@ metricsCollector:
 logsCollector:
   name: logs
   clusterName: ""
-  image: otel/opentelemetry-collector-contrib:0.103.0
+  image: otel/opentelemetry-collector-contrib:0.105.0
```

Comment:
I see some operator references to v0.103.0 in this file - should these be updated? https://github.com/lightstep/otel-collector-charts/blob/05ffb3716956b81efcb66608fcd088504830b647/charts/kube-otel-stack/values.yaml#L1255

Reply:
right now no, operator hasn't released 105 yet

Reply:
I just meant to match your change in otel-cloud-stack https://github.com/lightstep/otel-collector-charts/pull/89/files#diff-8ef2df2ac532c011dd5132a1df9e3e4b65d8847758a66974b209a2927486c906R750

Reply:
ah yeah same with the bridge (it's released w/ the operator) so we're kinda split rn

Reply:
Hmm okay not sure I completely follow but just wanted to point out that otel-cloud-stack is now on 0.104.0 while kube-otel-stack is on 0.103.0 of the `operator-opamp-bridge`

Reply:
ohhhhhhhhh 🤦 now i see